robot_ws/src/person_detection_robot/launch/monitoring.launch.py

```diff
@@ -0,0 +1,118 @@
+    default_aws_region = config_yaml['cloudwatch_metrics_collector']['ros__parameters']['aws_client_configuration']['region']
+
+    with open(default_logs_config, 'r') as f:
+      config_text = f.read()
+    config_yaml = yaml.safe_load(config_text)
+    default_log_group_name = config_yaml['cloudwatch_logger']['ros__parameters']['log_group_name']
+
+    default_aws_region = os.environ.get('ROS_AWS_REGION', default_aws_region)
+
+    launch_actions = [
+        launch.actions.DeclareLaunchArgument(
+            name='aws_region',
+            description='AWS region override, defaults to config .yaml if unset',
+            default_value=default_aws_region
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='launch_id',
+            description='Used for resource name suffix if specified',
+            default_value=launch.substitutions.EnvironmentVariable('LAUNCH_ID')
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='metrics_node_name',
+            default_value="cloudwatch_metrics_collector"
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='aws_metrics_namespace',
+            default_value=default_aws_metrics_namespace
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='logger_node_name',
+            default_value='cloudwatch_logger'
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='log_group_name',
+            default_value=default_log_group_name
+        ),
+        launch.actions.SetLaunchConfiguration(
+            name='aws_metrics_namespace',
+            value=PythonExpression(["'", LaunchConfiguration('aws_metrics_namespace'), "-", LaunchConfiguration('launch_id'), "'"]),
+            condition=IfCondition(PythonExpression(["'true' if '", LaunchConfiguration('launch_id'), "' else 'false'"]))
+        ),
+        launch.actions.SetLaunchConfiguration(
+            name='log_group_name',
+            value=PythonExpression(["'", LaunchConfiguration('log_group_name'), "-", LaunchConfiguration('launch_id'), "'"]),
+            condition=IfCondition(PythonExpression(["'true' if '", LaunchConfiguration('launch_id'), "' else 'false'"]))
+        ),
+        launch.actions.IncludeLaunchDescription(
+            launch.launch_description_sources.PythonLaunchDescriptionSource(
+                os.path.join(get_package_share_directory('health_metric_collector'), 'launch', 'health_metric_collector.launch.py')
+            ),
+            launch_arguments={
+                'config_file': os.path.join(get_package_share_directory('person_detection_robot'), 'config', 'health_metrics_config.yaml')
+            }.items()
+        ),
+        launch.actions.IncludeLaunchDescription(
+            launch.launch_description_sources.PythonLaunchDescriptionSource(
+                os.path.join(get_package_share_directory('cloudwatch_metrics_collector'), 'launch', 'cloudwatch_metrics_collector.launch.py')
```

Comment:
Similarly, no `cloudwatch_metrics_collector` dependency listed in `package.xml`

Reply:
Missed! Done change.